### PR TITLE
Convert from donut to integrant

### PR DESCRIPTION
**Pull request by cap10morgan**

### CODE REVIEW

**Comment by cap10morgan** [comment]
* **#4** <a href="https://app.graphite.dev/github/pr/fluree/kinesis-component/4?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 2  dependent PRs ([#5](https://github.com/fluree/kinesis-component/pull/5) <a href="https://app.graphite.dev/github/pr/fluree/kinesis-component/5?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#6](https://github.com/fluree/kinesis-component/pull/6) <a href="https://app.graphite.dev/github/pr/fluree/kinesis-component/6?utm_source=stack-comment-icon" target="_blank">Graphite</a>) 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @cap10morgan and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/fluree/kinesis/stream.clj [hunk 1]
```diff
@@ -1,57 +1,117 @@
...
-                                                          ListStreamsResponse)))
+           (software.amazon.awssdk.services.kinesis.model
+            CreateStreamRequest DescribeStreamSummaryRequest
+            StreamDescriptionSummary StreamStatus DescribeStreamSummaryResponse
+            PutRecordRequest ListStreamsRequest ListStreamsResponse
+            StreamSummary)))
+
+(set! *warn-on-reflection* true)
+
+(defmulti dejavaify-stream-summary (fn [pojo] (class pojo)))
```
Comment (dpetran) [inline src/fluree/kinesis/stream.clj:15]: : )

**Review by dpetran — APPROVED** [review]
:doughnut: :wave:

**Comment by cap10morgan** [comment]
### Merge activity

* **Jul 18, 5:29 PM EDT**: @cap10morgan started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/fluree/kinesis-component/4).
* **Jul 18, 5:29 PM EDT**: @cap10morgan merged this pull request with [Graphite](https://app.graphite.dev/github/pr/fluree/kinesis-component/4).

**Pull request merged**